woocommerce/woocommerce
# Coming soon: Load Cardo and Inter fonts only when site is not live

### Changes proposed in this Pull Request:

When the site is live, it should not include Cardo and Inter fonts on the frontend, because there is no reason to do so.

### How to test the changes in this Pull Request:

1. Have WooCommerce version with Coming soon feature enabled
2. Set the site to Coming Soon in WooCommerce > Settings > Site Visibility
3. Load the shop page, if you are logged in as an administrator in the site you'll need to log out or load the page in an incognito window to actually see the "Coming soon" page
4. Verify that Cardo and Inter fonts load along with inline CSS styles, you can see the fonts list in the browser inspector > Application > Frames > Font
5. Set the site to Live in WooCommerce > Settings > Site Visibility
6. Verify that Cardo and Inter fonts don't load and no CSS styles aren't added


### Changelog entry




<details>

<summary>Coming soon: Load Cardo and Inter fonts only when site is not live</summary>

#### Significance




#### Type




#### Message 
Load Cardo and Inter fonts only when site is not live reducing HTML size and potential network requests
</details>
